xbmc/ServiceBroker.cpp

```diff
@@ -16,30 +16,43 @@
 
 using namespace KODI;
 
-std::unique_ptr<CLog> CServiceBroker::m_logging;
+
+CServiceBroker::CServiceBroker() :
+    m_pGUI(nullptr),
+    m_pWinSystem(nullptr),
+    m_pActiveAE(nullptr),
```

Comment:
I seriously doubt that this needed. ActiveAE get created / registered on startup. On shutdown it gets unregistered and deleted. Making it a global is nonsense.

Reply:
It was already a global in ServiceBroker (a static member). I didn't change that. This change just fixes the deinitialization ordering problem with BSS segments. Feel free to move it out. None of these should actually be globals.